Add TargetFramework : netstandard1.3

StefH opened the pull request:
Added target framework **netstandard 1.3** to make it possible to use this library in netstandard 1.3 libraries.

springcomp commented on src/jmespath.net/jmespath.net.csproj:
```diff
@@ -1,10 +1,10 @@
     <AssemblyTitle>JmesPath.Net</AssemblyTitle>
     <Authors>Jeremie Devillard;Maxime Labelle</Authors>
-    <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
+    <TargetFrameworks>netstandard1.3;netstandard2.0;net45</TargetFrameworks>
```
Please, update `src/jmespath.net.signed/jmespath.net.signed.csproj` accordingly.

springcomp replied:
Please, update the corresponding `tests/jmespathnet.tests/jmespathnet.tests.csproj` with an additionnal target framework defined to `netcoreapp1.0`.

Also, add a corresponding trigger for this test configuration in `appveyor.yml` by adding a line in the `test_script` section of the file.

springcomp commented:
With the requested changes, this pull request would be candidate for integration after #12.
Please, review the #12 pull request as a guideline for the changes.

StefH commented:
I've added the new framework also to the signed project.
And updated the unit-test project.

Not need to update the appveyor for the unit-tests, both frameworks are tested. (See the appveyor build output)
